packages/p2p/src/components/modal-manager/modal-manager-context-provider.jsx

```diff
@@ -48,6 +48,14 @@ const ModalManagerContextProvider = props => {
         }, [modals]);
     };
 
+    /**
+     * Checks if the current visible modal matches the specified modal key passed to the argument.
+     * Can also be used to check for multiple modal keys.
+     *
+     * @param {...string} keys - the modal keys to check if the current visible modal matches it
+     */
+    const isCurrentModal = (...keys) => keys.includes(active_modal.key);
+
```

Comment:
Added helper methods from unmerged branch (which was already set to Ready status): https://github.com/binary-com/deriv-app/pull/7505/files